docs/code/transports/RabbitMqConsoleListener.cs

```diff
@@ -49,8 +50,7 @@ public static async Task Main()
             }
         }
 
-        class OrderSubmittedEventConsumer :
-            IConsumer<OrderSubmitted>
+        private sealed class OrderSubmittedEventConsumer : IConsumer<OrderSubmitted>
```

Comment:
yeah, no, I don't use explicit keywords when they're implicit. The `using` addition is great though, but I won't accept this with the default `private` keyword, even `sealed` is just noise.

Reply:
So you're saying that you are **explicitly** breaking the [IDE0040](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0040) rule huh?

Reply:
I prefer to think of it as following the `omit_if_default` option. They're all options, this is what I use.